tests/app/forms/test_questionnaire_form.py

```diff
@@ -124,8 +124,8 @@ def test_invalid_calculation_type(self):
             question_json = {
                 'id': 'breakdown-question',
                 'type': 'Calculated',
-                'calculated': {
-                    'calculated_type': 'subtraction',
+                'calculations': [{
+                    'calculation_type': 'subtraction',
```

Comment:
Whats this?

Reply:
So it appears @shahi645 added tests for scenarios he doesn't allow at a schema level and we let it through. @shahi645 do we need to add this at schema level or remove this test until we have a full set of tests for it?

Reply:
Ignore this. The test is actually ensuring it can handle an invalid calculation type.

Reply:
it is at schema level but Its good to test it here too, it was a PR change requested